### PR TITLE
Move create clm filters for RH-like minions out of the parallel loop

### PR DESCRIPTION
**Pull request by maximenoel8**
### What does this PR do : 

Run sequentially `Feature: Create the CLM filters for RH-like minions` .

### Explanation

I didn't change the custom repositories behavior. So far, the tags were protecting the run set to be executed several time.
But : 
Add custom repositories run set is currently executed in the parallel loop. In this run set, the feature : `Create the CLM filters for RH-like minions` is not protected by any tags and so will be executed for all the minions users.

I created a specific run set for this feature and this run set will be executed before the client stage. It's not very clean but I think a refactoring of custom repositories feature will be needed at one point. I didn't work on it because it was not generic.

### Uyuni PR :

### CODE REVIEW

File: jenkins_pipelines/environments/common/pipeline-build-validation.groovy [hunk 1]
```diff
@@ -311,81 +322,79 @@ def clientTestingStages() {
                         res_common_channels = sh(script: "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'unset ${temporaryList.join(' ')}; export BUILD_VALIDATION=true; cd /root/spacewalk/testsuite; rake cucumber:build_validation_add_common_channels'", returnStatus: true)
                         echo "Custom channels and MU repositories status code: ${res_common_channels}"
                         if (res_common_channels != 0) {
-                            if (params.must_run_containerization_tests) {
```
Comment (srbarrios) [inline jenkins_pipelines/environments/common/pipeline-build-validation.groovy:314]: I don' t get why we had this condition 🤔

**Review by srbarrios — CHANGES_REQUESTED** [review]
Small changes, but please change them before merging it.
If I'm not available when that's ready, feel free to merge it without my approval.

File: jenkins_pipelines/environments/common/pipeline-build-validation.groovy [hunk 2]
```diff
@@ -166,6 +166,17 @@ def run(params) {
 
             // Call the minion testing.
             try {
+                if (params.must_add_custom_channels) {
```
[the remark this answers is not in the document]
Comment (Bischoff) [reply jenkins_pipelines/environments/common/pipeline-build-validation.groovy:169]: Isn't that called instead "common channels" in the current stages?

**Review by Bischoff — COMMENTED** [review]
same as Oscar

**Pull request closed without merging**